### PR TITLE
refactor: neo collapse

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Related #5852
- [x] rename existing `NeoCollapse` to `NeoCollapsible`. extend oruga collapse to `NeoCollapse`

#### Did your issue had any of the "$" label on it?

- [ ] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=<My_Polkadot_Address_check_https://github.com/kodadot/nft-gallery/blob/main/REWARDS.md#creating-your-dot-address>)

## Screenshot 📸

- [ ] My fix has changed UI

### <samp>🤖 Generated by Copilot at 26e336a</samp>

This pull request introduces several changes to the UI components and the massmint feature of the NFT gallery. It replaces the `b-collapse` component with the `NeoCollapse` component from the `@kodadot1/brick` library, which has a more consistent and responsive design and supports icons. It also renames and refactors some components to better reflect their functionality and align with the new collection structure. It fixes some bugs and typos in the code, and adds keyboard shortcuts and collapsible filters for the user convenience. It deletes the `PriceChart.vue` component, which was no longer used, and removes some redundant or unnecessary code.

### <samp>🤖 Generated by Copilot at 26e336a</samp>

> _`NeoCollapse` shines_
> _A new component for filters_
> _Spring cleaning the code_

### CODE REVIEW

**Comment by prury** [comment]
did you change the colors in order to help me? ❤️ 

image


**_VERIFICATION PROCESS:_** 

massmint: 
UploadCompressedMedia.vue - ok 
UploadDescription.vue - ok 
OverviewTable.vue - ok

rmrk
Holder.vue - ok
Sales.vue - ok 


Search.vue - ok
KeyboardShortcuts.vue - ok 
Collapse Card Wrapper.vue
Collapse Wraper.vue
EvenTypeFilter.vue - ok 
Popular Collections.vue - ok
Price Filter.vue - ok 
Status Filter.vue - ok

will be finishing later today

**Comment by daiagi** [comment]
In future code
when should i use ```NeoCollapse``` and when to use ```NeoCollapsible```?

File: components/search/Search.vue [hunk 1]
```diff
@@ -44,27 +44,6 @@
...
-          label-color="has-text-success" />
-      </div>
-      <SearchPriceRange
-        v-if="!hideFilter && !isMoonRiver"
-        :range="priceRange"
-        @input="priceRangeChange"></SearchPriceRange>
-      <div v-if="priceRangeDirty" class="is-size-7">
-        <PriceRange inline />
-      </div>
-    </b-collapse>
```
Comment (daiagi) [inline components/search/Search.vue:67]: why removed?

File: components/rmrk/Gallery/PriceChart.vue
Comment (daiagi) [inline components/rmrk/Gallery/PriceChart.vue:1]: wasn't it used?

Comment (roiLeo) [reply components/rmrk/Gallery/PriceChart.vue:1]: not used

File: components/search/Search.vue (hunk 1 above)
Comment (roiLeo) [reply components/search/Search.vue:67]: old stuff, I don't think it's used anymore

**Review by roiLeo — APPROVED** [review]
✅ let it roll

**Comment by preschian** [comment]
> In future code when should i use `NeoCollapse` and when to use `NeoCollapsible`?

I don't have a strong opinion about this for now. I think if we have the same pattern we can extend variants on `NeoCollapsible`. otherwise, use `NeoCollapse` instead

**Comment by vikiival** [comment]
So was the color issue fixed?

**Comment by roiLeo** [comment]
> So was the color issue fixed?

Screenshot 2023-06-21 at 14-09-00 refactor neo collapse by preschian · Pull Request #6249 · kodadot_nft-gallery

fixed in #6298

**Comment by roiLeo** [comment]
> wasn't it #6264 ?

Nope if it's on canary then we merged it. (it was #6298 check #6269 conversation)

**Review by daiagi — APPROVED** [review]

**Pull request merged**